package.json

```diff
@@ -2,7 +2,7 @@
     "name": "vscode-edge-devtools",
     "displayName": "Microsoft Edge Tools for VS Code",
     "description": "Use the Microsoft Edge Tools from within VS Code to see your site's runtime HTML structure, alter its layout, fix styling issues as well as see your site's network requests.",
-    "version": "1.1.9",
+    "version": "1.1.10",

```

Comment:
I feel like the version update itself is big enough to update the minor to `1.2.0`.  Any thoughts on this?

Reply:
Seems reasonable to me. I'm not really sure what our versioning story is

Reply:
We haven't really fleshed it out too much (which we should), but I would want to somewhat follow these standards:

```
Given a version number MAJOR.MINOR.PATCH, increment the:

MAJOR version when you make incompatible API changes,
MINOR version when you add functionality in a backwards compatible manner, and
PATCH version when you make backwards compatible bug fixes.
```

Reply:
Updated (images in conversation show old version number, but they have been updated in code)